### PR TITLE
deleting some stuff

### PR DESCRIPTION
**Pull request by dosht**
url jira

Deleting some lines from the readme
- [x] deleting the lines
- [x] writing tests
- [x] ready for review and merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1 @@
 # github-workshop
-[x] ???
-* foo
-* bar
```
Comment (MarioAriasC) [inline README.md:4]: :hankey: What is this?? you should add something here!!

Comment (dosht) [inline README.md:4]: This is not supported anymore :dancer:

Comment (MarioAriasC) [inline README.md:4]: :angry: Change it!!

**Comment by MarioAriasC** [comment]
:+1:

**Pull request merged**